spec/index.html

```diff
@@ -335,8 +335,22 @@ <h3>Version Announcement</h3>
       -->
     </pre>
 
-    <p>When providing content over HTTP, servers can announce the version
-      using the optional `version` Content-Type parameter:</p>
+    <p>If the value has the form of an xsd:decimal, the surrounding
+      <a href="#cp-quotation-mark"><code title="quotation mark">&quot;</code></a>s
```

Comment:
'<a href="#cp-quotation-mark"><code title="quotation mark">&quot;</code></a>s' doesn't read well.
'quotation marks (<a href="#cp-quotation-mark"><code title="quotation mark">&quot;</code></a>)' reads better.
```suggestion
      quotation marks (<a href="#cp-quotation-mark"><code title="quotation mark">&quot;</code></a>)
```